src/app/auth/auth.component.ts

```diff
@@ -205,7 +210,12 @@ export class AuthComponent implements OnInit, OnDestroy {
 
   logIntoAnotherAccount() {
     this.electronService.clearCookies();
-    this.authService.startOAuthProcess();
+    try {
```

Comment:
I don't think we need to handle the error here, since `logIntoAnotherAccount` is called only in the Electron app, and the pop-up error is thrown only for the web app.

Reply:
Right, thanks for pointing it out😅